### PR TITLE
Remove benchmark command

### PR DESCRIPTION
**Pull request by felixarntz**
## Summary

This command is not directly related to the Performance Lab plugin and therefore doesn't really belong in this repository. It is instead being added to https://github.com/GoogleChromeLabs/wpp-research/pull/1.

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
     "enabled-modules": "./bin/plugin/cli.js enabled-modules",
-    "benchmark": "./bin/plugin/cli.js benchmark",
```
Comment (eugene-manuilov) [inline package.json:25]: We also need to remove `autocannon`, `table` and `csv-stringify` dependencies.

Comment (felixarntz) [reply package.json:25]: @eugene-manuilov Good catch, updated!

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks, @felixarntz, PR LGTM.

Commit [638816b](https://github.com/WordPress/performance/pull/569/commits/638816b4d752775a60b0966f5aa5ce6b6e2dc0cb#diff-fea647b6146d87b6e35c849da4276d9abc632a7f6e680d958819e4eea14ac6b4R9-R10) added `defaults` for the benchmark commands. It's not blocking if we don't remove it.

**Comment by felixarntz** [comment]
@mukeshpanchal27 Thanks!

> Commit [638816b](https://github.com/WordPress/performance/pull/569/commits/638816b4d752775a60b0966f5aa5ce6b6e2dc0cb#diff-fea647b6146d87b6e35c849da4276d9abc632a7f6e680d958819e4eea14ac6b4R9-R10) added `defaults` for the benchmark commands. It's not blocking if we don't remove it.

This doesn't need to be removed as it's a useful enhancement to keep.

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**